core/utils.h

```diff
@@ -2021,12 +2021,14 @@ notify(syslog_event_type_t priority, bool internal, bool synch,
 
 #define FATAL_ERROR_EXIT_CODE 40
 
-#define REPORT_FATAL_ERROR_AND_EXIT(dcontext, msg_id, arg_count, ...)               \
+#define REPORT_FATAL_ERROR_AND_EXIT(msg_id, arg_count, ...)               \
     do {                                                                            \
         /* Right now we just print an error message. In the future */               \
         /* it may make sense to generate a core dump too. */                        \
         SYSLOG_COMMON(false, SYSLOG_CRITICAL, msg_id, arg_count, __VA_ARGS__);      \
-        os_terminate_with_code(dcontext, TERMINATE_PROCESS, FATAL_ERROR_EXIT_CODE); \
+        /* We hard-code NULL for the dcontext because we know it isn't used */      \
+        /* with TERMINATE_PROCESS or our specific error code */                     \
```

Comment:
nit: missing trailing period, and normally you'd have no closing / and re-opening / for a multi-line single comment.

Reply:
Done